doc/ref_model/chapters/chapter03.rst

```diff
@@ -146,8 +146,8 @@ A virtual machine and container can consume storage through a number of means. T
 - managed via the hypervisor and container runtime (Hypervisor Attached for virtual machine and Container Persistent for
   containers) and is connected via cloud infrastructure underlay network and
 - Shared File Storage and the Object storage which is connected via the tenant / user overlay network.
-  The details of the tenant storage consumption model are covered in section 
-  :ref:`Storage for Tenant Consumption`.
+  The details of the tenant storage consumption model are covered in section
+  `Storage for Tenant Consumption`_.
```

Comment:
@collivier  Just for my benefit. Why the :ref: is not needed in this case and why it is needed in line 1062?